### PR TITLE
Elastic search set default DateTime format to epoch milliseconds

### PR DESCRIPTION
**Pull request by MosheEichler**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-hq.paloaltonetworks.local/browse/XSUP-27285

## Description
Do to BC. changed the default DateTime in ES to 'Timestamp-Milliseconds'.

## See here:
https://stackoverflow.com/questions/69210359/elastic-search-failed-to-parse-date-field-with-format-strict-date-optional-time#answer-70269810

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Elasticsearch/ReleaseNotes/1_3_14.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Elasticsearch v2
+Changed the default **Time field type** integration parameter to 'Timestamp-Milliseconds'.
```
Comment (ShirleyDenkberg) [inline Packs/Elasticsearch/ReleaseNotes/1_3_14.md:4]:
```suggestion
Changed the default *Time field type* integration parameter to *Timestamp-Milliseconds*.
```

**Comment by ShirleyDenkberg** [comment]
@dorschw @bziser Doc review completed.

**Review by bziser — APPROVED** [review]
great, just update docker

**Comment by JeromeSola** [comment]
Hello ,

First i'm sorry i'm not sure to put this message to the right place.
Let me know if i can put my message somewhere else.

We are using Cortex XSOAR and the Elasticsearch content pack with Elasticsearch_v2 integration.
However since version 1.3.13 my integration can't fetch incident anymore.
We've got the following error:
"Exception: Failed to execute the fetch-incidents command. Make sure the `Time field type` is correctly set."

My elasticsearch timestamp field looks like this : "2022-12-22T13:20:24.971Z"
In this case i still think that the Time field type should remain "Simple-date"

So I tried to understand your code and i copied the integration to modify the code to update the variable named: "DEFAULT_DATETIME_FORMAT" to put the following value: DEFAULT_DATETIME_FORMAT = 'YYYY-MM-DDTHH:mm:ssZ'.
Doing this works to fetch incidents in our case

So I wonder if you shouldn't put a variable to edit in the instance configuration to let the user put custom value to the DEFAULT_DATETIME_FORMAT.

I will rollback to version 1.3.12 until you fix this.

Thanks in advance for help

**Pull request merged**